pkg/cloudsql/collectors.go

```diff
@@ -0,0 +1,106 @@
+package cloudsql
+
+import (
+	"context"
+
+	"github.com/dbtuneai/agent/pkg/agent"
+	"github.com/dbtuneai/agent/pkg/metrics"
+	"github.com/sirupsen/logrus"
+)
+
+func CloudSQLHardwareInfo(logger *logrus.Logger, config Config, cloudMonitoringClient *CloudMonitoringClient) func(ctx context.Context, metric_state *agent.MetricsState) error {
```

Comment:
This function was a little dense, but you can choose either `io_ops_count` instead *OR* `IOPS`. I don't think we want to store both, as it's fairly redundant to do so.

NOTE: `IOPS` is not the same as `IO_OPS`, former is per-second, latter is total count. We had some issues relating to this before. Seems you're aware but just incase

Reply:
sure, no harm